.gitignore

```diff
@@ -7,6 +7,7 @@
 .platformio
 .pioenvs
 .piolibdeps
+.pio/libdeps
```

Comment:
Why do you need this one?

Reply:
hmmm. I think I needed this because at some point I used a dev branch of IRremote and PIO would create some temp files in .pio/libdeps .
It seems the `.piolibdeps` folder (one line above in current file) has been replaced with `.pio/libdeps` in newer versions of PIO.
https://community.platformio.org/t/piolibdeps-folder/8830
we can remove it from here and add in another PR or somebody on the team can add it. whatever you prefer.